tests/dependency/test_dependency.py

```diff
@@ -5,16 +5,8 @@
 
 import numpy as np
 import pytest
+import substra
 from pydantic import ValidationError
-from substra import BackendType
-from substra.sdk.schemas import AlgoInputSpec
-from substra.sdk.schemas import AlgoOutputSpec
-from substra.sdk.schemas import AlgoSpec
-from substra.sdk.schemas import AssetKind
-from substra.sdk.schemas import CompositeTraintupleSpec
-from substra.sdk.schemas import ComputeTaskOutputSpec
-from substra.sdk.schemas import InputRef
-from substra.sdk.schemas import Permissions
```

Comment:
Small question: is this a good practice to take into account in the future? (limit the number of import)

Reply:
yes, it is better to import the module and get the fields from it
https://google.github.io/styleguide/pyguide.html#22-imports

Reply:
it also improves readability, in the code you see that the class you are using is `schemas.Permissions`, that it comes from an imported module and not sthg defined somewhere in the file